tests/rptest/tests/cluster_features_test.py

```diff
@@ -339,8 +339,7 @@ def test_upgrade(self):
             f"downgraded logical version {initial_version}"
 
         # Restart nodes one by one.  Version shouldn't increment until all three are done.
-        self.installer.install([self.redpanda.nodes[0]],
-                               RedpandaInstaller.HEAD)
+        self.installer.install([self.redpanda.nodes[0]], (22, 3))
```

Comment:
Why use (22,2) for the tests above, but (22,3) here?

Reply:
Ok, I think these are failing because 23.1.x doesn't yet have a higher logical version number than 22.3.x, because we didn't yet add anything new with a feature flag since releasing 22.3.x.

It's a bit artificial, but it probably make sense to just increment the logical version now (i.e. in `feature_table.cc` `latest_version` to 9).  Then in future we should remember to do the same thing when we add the `-dev` tag, so that these tests keep making sense.

Reply:
Ok so i'll mark the test ok to fail until the logical version is updated, and I'll open an issue if needed to keep track of this

Reply:
No, you can just bump the version in this PR.  Let's not mark anything ok_to_fail.

Reply:
fair enough. i will also update this test to dynamically fetch release, previous release, current logical version, previous logical version

Reply:
should also add an entry in feature_schema?